lib/twitter_web/templates/page/index.html.eex

```diff
@@ -1,38 +1,14 @@
 <section class="phx-hero">
-  <h1><%= gettext "Welcome to %{name}!", name: "Phoenix" %></h1>
-  <p>Peace of mind from prototype to production</p>
+  <h1><%= gettext "Welcome, %{name}!", name: "Rap Birds" %></h1>
+  <p>This is your poetic twitter. <br> Share your wit-twit!</p>
 </section>
 
-<section class="row">
+<!--<section class="row">
```

Comment:
If this is a comment lets delete unless theres a good reason to include

Reply:
ok!